capnslog/README.md

```diff
@@ -0,0 +1,41 @@
+# CoreOS Log
+
+There are far too many logging packages out there, with varying degrees of licenses, far too many features (colorization, all sorts of log frameworks) or are just a pain to use (lack of `Fatalln()`?)
+
+## Design Principles
+
+* Logging is off by default, except critical failures, which are unrecoverable.
```

Comment:
I'd like to see the default level at INFO. Logs aren't helpful if they're getting dumped on the floor.